icu4c/source/extra/scrptrun/scrptrun.cpp

```diff
@@ -43,11 +43,7 @@ UChar32 ScriptRun::pairedChars[] = {
     0x301a, 0x301b
 };
 
-const int32_t ScriptRun::pairedCharCount = UPRV_LENGTHOF(pairedChars);
-const int32_t ScriptRun::pairedCharPower = 1 << highBit(pairedCharCount);
-const int32_t ScriptRun::pairedCharExtra = pairedCharCount - pairedCharPower;
-
-int8_t ScriptRun::highBit(int32_t value)
+static constexpr int8_t highBit(int32_t value)
```

Comment:
@markusicu I tried to minimize the changes to this file, but this one time use function really should be switched to a constexpr function (C++11 syntax) to avoid the C++ static initialization. The constexpr functions are hard to separate between definition and declaration.